streamz/dataframe/core.py

```diff
@@ -698,16 +733,21 @@ def _accumulate(self, Agg, **kwargs):
                                       diff=diff,
                                       window=window)
 
-        for typ, s_type in _stream_types[stream_type]:
-            if isinstance(example, typ):
+        for _, s_type in _stream_types[stream_type]:
+            if is_dataframe_like(example):
+                return s_type(outstream, example)
+            elif is_series_like(example):
+                return s_type(outstream, example)
+            elif is_index_like(example):
```

Comment:
I'm not sure that I understand this code change.  

It looks like we always use the first `s_type` found, regardless of the type of the `example`

Reply:
I think it's a mistake, will fix this. I thought I am going to get away with not having to reference pandas and cudf classes explicitly in _stream_types.